cmd/worker/option/option.go

```diff
@@ -81,6 +83,8 @@ func (a *Worker) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&a.MysqlConnectionInfo, "mysql", "root:admin@tcp(127.0.0.1:3306)/region", "mysql db connection info")
 	fs.StringSliceVar(&a.EventLogServers, "event-servers", []string{"127.0.0.1:6366"}, "event log server address. simple lb")
 	fs.StringVar(&a.KubeConfig, "kube-config", "", "kubernetes api server config file")
+	fs.IntVar(&a.KubeAPIQPS, "kube-api-qps", 50, "kube client qps")
```

Comment:
Does this value need to be intelligently adjusted for the cluster situation?

Reply:
There are many factors that affect qps, such as the number of kube-apiserver instances, the CPU and memory of kube-apiserver, ETCD, and the number of worker instances, etc. It's hard to adjust the qps intelligently